src/lsp/client/client.ts

```diff
@@ -0,0 +1,204 @@
+import { provideSignatureHelp } from '../../providers/signature';
+import { isResultsDoc } from '../../results-output/results-doc';
+import * as vscode_lsp from 'vscode-languageclient/node';
+import * as downloader from './downloader';
+import * as config from '../../config';
+import * as messages from './messages';
+import * as vscode from 'vscode';
+import * as path from 'path';
+
+// class ExecuteCommandDisablement implements vscode_lsp.DynamicFeature<any> {
```

Comment:
I'm leaving this in because I think it might be useful if someone ever wants to revisit this problem.

Reply:
Let's put a comment above this code stating this. I tend to like to remove commented code, partly because git can be used to get the past version of the code, and partly to have a cleaner code base. I think it's fine to leave for now, but a comment above stating why would help ensure no one deletes it later without knowing why it was left.

Reply:
Done.